cli/packages/cmd/login.go

```diff
@@ -342,16 +409,16 @@ func getFreshUserCredentials(email string, password string) (*api.GetLoginOneV2R
 	return &loginOneResponseResult, &loginTwoResponseResult, nil
 }
 
-func shouldOverrideLoginPrompt(currentLoggedInUserEmail string) (bool, error) {
+func userLoginMenu(currentLoggedInUserEmail string) (bool, error) {
 	prompt := promptui.Select{
-		Label: fmt.Sprintf("There seems to be a user already logged in with the email: %s. Would you like to override that login? Select[Yes/No]", currentLoggedInUserEmail),
-		Items: []string{"No", "Yes"},
+		Label: fmt.Sprintf("Current logged in user email: %s", currentLoggedInUserEmail),
```

Comment:
what do you think about also adding which host they are on such as `current logged in user email: maidul@ss.com on domain: app.infisical.com`?

Reply:
Yeah this shouldn't be an issue, there is an added check for the current logged in user domain in https://github.com/Infisical/infisical/pull/481/files/be2cf54d6e138b8d82a5310be941f43310b9823f#diff-eb3a7252b09a5793a44fec4d5e033dda44f2fc86484c3d1a728bb57208058fda.